common/fee/models.go

```diff
@@ -0,0 +1,100 @@
+package fee
+
+import (
+	"math/big"
+
+	"github.com/pkg/errors"
+
+	"github.com/smartcontractkit/chainlink/v2/common/chains/label"
+	"github.com/smartcontractkit/chainlink/v2/core/logger"
+	bigmath "github.com/smartcontractkit/chainlink/v2/core/utils/big_math"
+)
+
+var (
+	ErrBumpFeeExceedsLimit = errors.New("fee bump exceeds limit")
+	ErrBump                = errors.New("fee bump failed")
+	ErrConnectivity        = errors.New("transaction propagation issue: transactions are not being mined")
+)
+
+func IsBumpErr(err error) bool {
+	return err != nil && (errors.Is(err, ErrBumpFeeExceedsLimit) || errors.Is(err, ErrBump) || errors.Is(err, ErrConnectivity))
+}
+
+// CalculateFee computes the fee price and chain specific fee limit for a transaction.
+func CalculateFee(
+	maxFeePrice, defaultPrice, maxBumpPrice *big.Int,
+) (feePrice *big.Int) {
+	feePrice = CapFeePrice(defaultPrice, maxFeePrice, maxBumpPrice)
+	return
+}
+
+// CalculateBumpedFee will increase the price and apply multiplier to the fee limit.
+func CalculateBumpedFee(
+	lggr logger.SugaredLogger,
+	currentFeePrice, originalFeePrice, maxFeePrice,
+	maxBumpPrice, bumpMin *big.Int,
+	bumpPercent uint16,
+	toChainUnit func(*big.Int) string,
+) (*big.Int, error) {
+	feePrice, err := bumpFeePrice(lggr, currentFeePrice, originalFeePrice, maxFeePrice, maxBumpPrice, bumpMin, bumpPercent, toChainUnit)
+	if err != nil {
+		return nil, err
+	}
+	return feePrice, nil
```

Comment:
Could return directly?
```suggestion
	return bumpFeePrice(lggr, currentFeePrice, originalFeePrice, maxFeePrice, maxBumpPrice, bumpMin, bumpPercent, toChainUnit)
```
Is there more room for simplification here? Do we actually need two different funcs?